### DISCUSSION TITLE
Game crashes when i start a world

### DISCUSSION THREAD

**Discussion by ghost** [MoreMcmeta/core#95] [Help Wanted]
I downloaded this mod and also download an texturepack called NewGlowing Ores. When i start the launcher the game load with normality, but when i want to play in my world, this load and the game crashes just after to the load barr is 100%. Can u help me?

This is the crash report: [crash-2024-08-12_20.31.08-client.txt](https://github.com/user-attachments/files/16593411/crash-2024-08-12_20.31.08-client.txt)

**Comment by soir20** [comment]
Can you confirm whether this happens when you're only using MoreMcmeta and the MoreMcmeta emissive plugin with the New Glowing Ores resource pack? I suspect it's a compatibility issue with another mod.

**Reply by ghost** [reply]
I dont have other texture pack to prove it, NewGlowingOres is the only texture pack for a while that ask the mod MoreMcmeta and also the plugin. When i dont use the texture pack the game works fine. I dont know, i really like this texture. In the 1.20.6 i havent issues

**Reply by ghost** [reply]
I been testing different texture pack and the game still crashing. What mod is generating issues? 
This the crash report with other texture pack and other mod called Continuity [crash-2024-08-13_10.22.05-client.txt](https://github.com/user-attachments/files/16601824/crash-2024-08-13_10.22.05-client.txt)

**Reply by soir20** [reply]
> What mod is generating issues?

It is difficult to tell from the crash report. Can you please try creating a world with only MoreMcmeta and MoreMcmeta emissive plugin as your installed mods?

**Reply by ghost** [reply]
I do that, and it works, the issue happen when i run the texturepack ;C

**Reply by soir20** [reply]
I'm not asking about removing the texture pack--I'm asking about removing the other mods you're using
